src/pages/Facility/settings/organizations/components/FacilityOrganizationSelector.tsx

```
@@ -407,71 +600,57 @@ export default function FacilityOrganizationSelector(
               </>
             ) : (
-              <Popover open={open} onOpenChange={handleOpenChange}>
+              <Popover
+                open={open}
+                onOpenChange={handleOpenChange}
+                modal={false}
+              >
                 <PopoverTrigger asChild>
                   <Button
                     variant="outline"
                     role="combobox"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<select>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)